src/Classes/Options/Particles/Shape/ImageShape.ts

```diff
@@ -0,0 +1,32 @@
+import {IImageShape} from "../../../../Interfaces/Options/Shape/IImageShape";
+
+export class ImageShape implements IImageShape {
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new replaceColor
+     */
+    public get replace_color(): boolean {
+        return this.replaceColor;
+    }
+
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new replaceColor
+     * @param value
+     */
+    public set replace_color(value) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'value' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)